crypto/fipsmodule/service_indicator/service_indicator_test.cc

```diff
@@ -1598,61 +1598,119 @@ static const struct HKDFTestVector {
     },
 };
 
-// TODO(CryptoAlg-1281): Do we need to test HKDF_Expand separately from this?
+// Index into the kHKDFTestVectors array; used in the EVP_HKDF_Extract and
+// EVP_HKDF_Expand tests, below.
+#define EVP_HKDF_TEST_EXTRACT_EXPAND 3
+
 class HKDF_ServiceIndicatorTest : public TestWithNoErrors<HKDFTestVector> {};
 
 INSTANTIATE_TEST_SUITE_P(All, HKDF_ServiceIndicatorTest,
                          testing::ValuesIn(kHKDFTestVectors));
 
 TEST_P(HKDF_ServiceIndicatorTest, HKDFTest) {
-  const HKDFTestVector &test = GetParam();
```

Comment:
The rest of the indentation in this file is 2 columns withing functions and if statements. It was increased to 4 to some parts here.

Reply:
Except where it's not. 🤷 De-indenting my parts.